include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp

```diff
@@ -203,7 +203,10 @@ class Orbit : public Trajectory
     String generateFrameName(const Orbit::FrameType& aFrameType) const;
 
```

Comment:
```suggestion

    /// @brief Find the Instant at which the return value of `getValue` crosses zero.
    ///
    ///              Use a bisection search to find the Instant between `previousInstant` and `nextInstant` at which the return value of `getValue` crosses zero. 
    ///
    /// @param anEpoch An orbit epoch from which to measure
    /// @param previousInstant lower bound of search
    /// @param getValue function that receives a duration [s] from `anEpoch` and returns a real value
    /// @return Instant
```

^nit: it's not super obvious what the extra Instant is for